libs/types/src/global-types-mappings.ts

```diff
@@ -401,7 +401,8 @@ export const TransferTypeMapping: TransferTypeMap = {
   TRANSFER_TYPE_WITHDRAW: 'Withdrawal',
   TRANSFER_TYPE_DEPOSIT: 'Deposit',
   TRANSFER_TYPE_BOND_SLASHING: 'Bond slashed',
-  TRANSFER_TYPE_STAKE_REWARD: 'Rewards funded',
+  // TRANSFER_TYPE_STAKE_REWARD: 'Rewards funded',
```

Comment:
chore (blocking): can we tidy up this unused code and make sure the correct map keys are in place